comp/website/live-mode.yaml

```diff
@@ -0,0 +1,30 @@
+summary: Switch the website (proxy) to Competition Live mode for the $SRYYYY competition
+
+priority: should
+
+component: Website
+
+milestone: $SRYYYY Competition Day
+
+area-owner: media-press
+
+description: >-
+  There may be website alterations to be made to the competition website
+  before the competition, such as adding an embed to the livestream.
+
+  During the competition, the website proxy should be switched to use
+  the competition live homepage instead of the normal one.
+  This normally happens on the morning of the competition.
+
+
+  After the competition, the website proxy should be switched back to using
+  the normal website homepage instead of the competition live one.
+  This normally happens on the morning following the competition.
+
+  Required actions:
+   - [ ] Add an embed to the livestream
+   - [ ] Switch the website proxy to use the competition live homepage
```

Comment:
Suggestion: Perhaps worth noting the required change? https://github.com/srobo/ansible/blob/main/group_vars/all.yml#L23